src/admin.rs

```diff
@@ -151,10 +155,13 @@ impl PgqrsAdmin {
 
     /// Verify that pgqrs installation is valid and healthy.
     ///
+    /// # Arguments
+    /// * `schema_name` - The name of the schema to verify
+    ///
     /// # Returns
     /// Ok if installation is valid, error otherwise.
-    pub async fn verify(&self) -> Result<()> {
-        let schema_exists_statement = SCHEMA_EXISTS_QUERY.replace("{PGQRS_SCHEMA}", PGQRS_SCHEMA);
+    pub async fn verify(&self, schema_name: &str) -> Result<()> {
```

Comment:
The functionality has to change. Check   the following:
* QueueRepository exists
* WorkerRepository exists
* All queues in queue repository have tables for queue and archive.